.github/workflows/set_rook_ceph_cluster.yaml

```diff
@@ -0,0 +1,52 @@
+name: rook-ceph-cluster
+on:
+  pull_request:
+    branches:
+      - main
+      - release-*
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+# cancel the in-progress workflow when PR is refreshed.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+  cancel-in-progress: true
+
+jobs:
+  rook-ceph-cluster:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+            fetch-depth: 0
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+
+      - name: set up tmate session for debugging
+        if: ${{ failure() || runner.debug || contains(github.event.pull_request.labels.*.name, 'debug-ci') }}
+        uses: mxschmitt/action-tmate@v3
+        with:
+          limit-access-to-actor: false
+          detached: true
+
+
+      - name: Setup Minikube
+        run: |
+            test/scripts/github-action-helper.sh install_minikube_with_none_driver v1.28.4
```

Comment:
set up the latest released version of kubernetes and also make the minikube_version as configurable here so we don't need to look into multiple places to change the configurations

Reply:
done